src/services/services.ts

```diff
@@ -1208,6 +1208,7 @@ namespace ts {
         textSpan: TextSpan;
         fileName: string;
         isWriteAccess: boolean;
+        isDefinition?: boolean;

```

Comment:
I made it required. Turns out that the deprecated `getOccurrencesAtPosition` didn't set it, so I set it to false in that case.